app/javascript/hyacinth_v1/components/digital_objects/new/DigitalObjectNewForm.js

```diff
@@ -30,15 +30,14 @@ function DigitalObjectNewForm() {
     descriptiveMetadata: {},
     identifiers: [],
   };
-
   return (
     <>
       <ContextualNavbar
         title={`New ${capitalize(initialDigitalObject.digitalObjectType)}`}
         rightHandLinks={[{ link: '/digital_objects', label: 'Back to Digital Objects' }]}
       />
       <DigitalObjectSummary digitalObject={initialDigitalObject} />
-      <MetadataForm formType="new" digitalObject={initialDigitalObject} />
+      <MetadataForm formType="new" digitalObject={initialDigitalObject} project={project} createType={digitalObjectType} />
```

Comment:
Instead of passing in project and createType, are you able to internally use (within the MetadataForm) the primaryProject and digitalObjectType fields of the passed-in digitalObject?

Reply:
Possibly - need to review what's in the stub object and how the new/edit modes differ.

Reply:
Removed redundant parameters.